### PR TITLE
Fix/regration-fixes-24.3

### PR DESCRIPTION
**Pull request by leon-up9**

### CODE REVIEW

**Comment by IgorGov** [comment]
@leon-up9 can you please elaborate what this PR is actually fixes

File: ui-common/src/components/OasModal/OasModal.tsx [hunk 1]
```diff
@@ -42,7 +44,7 @@ const OasModal = ({ openModal, handleCloseModal, getOasServices, getOasByService
         const data = await getOasByService(selectedService);
         setSelectedServiceSpec(data);
       } catch (e) {
-        toast.error("Error occurred while fetching service OAS spec");
+        toast.error("Error occurred while fetching service OAS spec",{containerId: TOAST_CONTAINER_ID});
```
Comment (RamiBerm) [inline ui-common/src/components/OasModal/OasModal.tsx:47]: format file

File: ui-common/src/components/TrafficViewer/EntryDetailed.tsx [hunk 2]
```diff
@@ -125,7 +126,7 @@ export const EntryDetailed = () => {
                         pauseOnHover: true,
                         draggable: true,
                         progress: undefined,
-                    });
+                    },{containerId: TOAST_CONTAINER_ID});
```
Comment (RamiBerm) [inline ui-common/src/components/TrafficViewer/EntryDetailed.tsx:129]: format file

File: ui-common/src/components/TrafficViewer/TrafficViewer.tsx [hunk 3]
```diff
@@ -194,7 +194,7 @@ export const TrafficViewer : React.FC<TrafficViewerProps> = ({setAnalyzeStatus,
             pauseOnHover: true,
             draggable: true,
             progress: undefined,
-          });
+          },{containerId: TOAST_CONTAINER_ID});
```
Comment (RamiBerm) [inline ui-common/src/components/TrafficViewer/TrafficViewer.tsx:197]: format file

File: ui/src/components/ServiceMapModal/ServiceMapModal.tsx [hunk 4]
```diff
@@ -99,7 +100,7 @@ export const ServiceMapModal: React.FC<ServiceMapModalProps> = ({ isOpen, onOpen
     const commonClasses = useCommonStyles();
     const [isLoading, setIsLoading] = useState<boolean>(true);
     const [graphData, setGraphData] = useState<GraphData>({ nodes: [], edges: [] });
-
+
```
Comment (RamiBerm) [inline ui/src/components/ServiceMapModal/ServiceMapModal.tsx:103]: remove this space

File: ui/src/components/ServiceMapModal/ServiceMapModal.tsx [hunk 5]
```diff
@@ -140,7 +141,7 @@ export const ServiceMapModal: React.FC<ServiceMapModalProps> = ({ isOpen, onOpen
             setGraphData(newGraphData)
 
         } catch (ex) {
-            toast.error("An error occurred while loading Mizu Service Map, see console for mode details");
+            toast.error("An error occurred while loading Mizu Service Map, see console for mode details",{containerId: TOAST_CONTAINER_ID});
```
Comment (RamiBerm) [inline ui/src/components/ServiceMapModal/ServiceMapModal.tsx:144]: format file

**Review by RamiBerm — CHANGES_REQUESTED** [review]

**Review by RamiBerm — APPROVED** [review]

**Pull request merged**